ccsds/PacketTypes.asn1

```diff
@@ -46,11 +45,11 @@ PacketVersionNumberValue ::= NULL
 Packet-ID {PacketType} ::= SEQUENCE
 {
     packetType PacketType,
-    secondaryHeaderFlag BOOLEAN,
+--! secondaryHeaderFlag INTEGER (0 .. 1), --! moved to ACN
     applicationProcess-ID ApplicationProcess-ID
 }
 
-SecondaryHeaderFlag ::= BOOLEAN
+SecondaryHeaderFlag ::= INTEGER (0 .. 1)
```

Comment:
just a suggestion, maybe `INTEGER(0, 1)` would be more readable. I think (but I'm not sure) you could even do a `INTEGER(headerDisabled(0), headerEnabled(1))`